### DISCUSSION TITLE
Some doubts about this project

### DISCUSSION THREAD

**Discussion by Greatz08** [BepInEx/BepInEx#1021] [General]
`BepInEx is a plugin / modding framework for Unity Mono, IL2CPP and .NET framework games (XNA, FNA, MonoGame, etc.)`

Is this project used for modding/hacking values in unity based games ? if yes then is this legal ? if no then what does it mean by modding framework ?

**Comment by toebeann** [comment]
It's for loading mods in games. Do you know what mods are? You can browse https://nexusmods.com and https://thunderstore.com for some example if you don't.

**Reply by Greatz08** [reply]
oo that type of mods, now i get it. I was thinking about different type of mod
